src/main/java/foodwhere/ui/CopyMenuItem.java

```diff
@@ -98,6 +103,7 @@ public CopyMenuItem(String tag, T item, Clipboard clipboard, Action action) {
     }
 
     private static void throwRunTimeError() {
+        logger.info("Invalid type of object passed in.");
```

Comment:
This would be more useful if it also logs `tag`, `item`, and `action`, since it happens alongside a `RuntimeException`.

Reply:
Resolved in latest commit